ganttproject/src/main/java/net/sourceforge/ganttproject/calendar/CalendarEditorPanel.java

```diff
@@ -303,12 +304,30 @@ public Component getTableCellRendererComponent(
     AbstractTableAndActionsComponent<CalendarEvent> tableAndActions = new AbstractTableAndActionsComponent<>(table) {
       @Override
       protected void onAddEvent() {
-        int lastRow = tableModel.getRowCount() - 1;
-        Rectangle cellRect = table.getCellRect(lastRow, 0, true);
-        table.scrollRectToVisible(cellRect);
-        table.getSelectionModel().setSelectionInterval(lastRow, lastRow);
-        table.editCellAt(lastRow, 0);
-        table.getEditorComponent().requestFocus();
+        LocalizedString title = InternationalizationKt.getRootLocalizer().create("add");
```

Comment:
Please create `calendar.editor.datePickerDialog.title` key in `biz.ganttproject.app.localization/translations/i18n.properties` file

Reply:
The key has been created in i18n only